vncviewer/Viewport.h

```diff
@@ -111,7 +111,8 @@ class Viewport : public Fl_Widget, public EmulateMB {
   PlatformPixelBuffer* frameBuffer;
 
   rfb::Point lastPointerPos;
-  uint8_t lastButtonMask;
+  uint16_t mouseButtonMask;
```

Comment:
This grouping can be a bit confusing. It has nothing to do with `lastPointerPos` or `lastButtonMask`.